content/docs/1.6.0/advanced-resources/backing-image-backup.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Backing Image Backup
+weight: 4
+---
+
+As of v1.6.0, Longhorn supports backing up of backing images.
+
+## Prerequisites
+
+You must first set up a backup target. For more information, see [Setting a Backup Target](../snapshots-and-backups/backup-and-restore/set-backup-target). Without setting up the backup target, Longhorn will report backup target unavailable and won't be able to back up the backing image.
+
+## Create and Restore a Backup of a Backing Image
+
+Since Backing Image is globally unique in Lnoghorn system, its Backup is also globally unique with the same name.
+
+You can create a backing image backup by YAML. Here is the example:
```

Comment:
```suggestion
## Create a Backup of a Backing Image

Because backing images are globally unique within the Longhorn system, the corresponding backups are also globally unique and are identified using the same name.

You can create a backups of backing images using YAML. 
```

Reply:
It seems like this suggestion was missed.